### PR TITLE
EZEE-3280: Added reset input .ez-data-source__destination-source-id when user remove image from image asset field

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-3280
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/fieldType/ezimageasset.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
     const SELECTOR_FIELD = '.ez-field-edit--ezimageasset';
     const SELECTOR_INPUT_FILE = 'input[type="file"]';
     const SELECTOR_INPUT_DESTINATION_CONTENT_ID = '.ez-data-source__destination-content-id';
+    const SELECTOR_INPUT_SOURCE_ID = '.ez-data-source__destination-source-id';
```
Comment (ViniTou) [inline src/bundle/Resources/public/js/scripts/fieldType/ezimageasset.js:5]: admin ui should not know about source field as it is EE only.

File: src/bundle/Resources/public/js/scripts/fieldType/ezimageasset.js [hunk 2]
```diff
@@ -209,6 +210,7 @@
             super.resetInputField();
 
             this.inputDestinationContentId.value = '';
+            this.inputSourcetId.value = '';
```
Comment (ViniTou) [inline src/bundle/Resources/public/js/scripts/fieldType/ezimageasset.js:213]: seems like a type.

**Comment by dew326** [comment]
Seems like the PR should in DAM connector repo?

**Comment by lucasOsti** [comment]
outdated

**Pull request closed without merging**